website/siteConfig.js

```diff
@@ -5,156 +5,10 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-// List of projects/orgs using your project for the users page.
```

Comment:
Thanks for the fix! Let's save it as `users.json` instead so that we don't need an extra yaml loader.